src/cryojax/simulator/_optics.py

```diff
@@ -17,214 +16,176 @@
     Constant,
 )
 from ..coordinates import cartesian_to_polar
-from ..typing import Real_, RealImage, Image, ComplexImage, ImageCoords
+from ..typing import Real_, RealImage, ComplexImage, Image, ImageCoords
 
 
 class CTF(AbstractFourierOperator, strict=True):
-    """
-    Compute a Contrast Transfer Function (CTF).
-
-    See ``cryojax.simulator.compute_ctf`` for more
-    information.
-
-    Attributes
-    ----------
-    degrees :
-    defocus_u :
-    defocus_v :
-    defocus_angle :
-    voltage :
-    spherical_aberration :
-    amplitude_contrast_ratio :
-    phase_shift :
+    """Compute the Contrast Transfer Function (CTF) in for a weakly
+    scattering specimen.
+
+    **Attributes:**
+
+    `defocus_u`: The major axis defocus in Angstroms.
+
+    `defocus_v`: The minor axis defocus in Angstroms.
+
+    `astigmatism_angle`: The defocus angle.
+
+    `voltage_in_kv`: The accelerating voltage in kV.
+
+    `spherical_aberration_in_mm`: The spherical aberration coefficient in mm.
+
+    `amplitude_contrast_ratio`: The amplitude contrast ratio.
+
+    `phase_shift`: The additional phase shift.
+
+    `degrees`: Whether or not the `astigmatism_angle` and `phase_shift` are given
+              in degrees or radians.
     """
 
     defocus_u: Real_ = field(default=10000.0, converter=jnp.asarray)
```

Comment:
`defocus_u/v_in_angstroms`

Reply:
Also wasn’t sure about this… was tempted to keep things similar to relion and CTFFIND, where I think defocus is always assumed to be measured in angstroms?

Reply:
looked more into this, looks like ctffind has both “defocus1_in_angstroms” and “defocus1” (which may not be in angstroms? a little confused about this). but changed names to defocus_u/v_in_angstroms